_posts/2019-07-27-template - with a long sample title.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I don't think we need more templates - it's simply to demonstrate how to create a new post.
There's nothing to stop you from using it locally, but no reason to have it committee to the repository, I don't think.

Reply:
File removed